keyvault/client/secrets/README.md

```diff
@@ -0,0 +1,309 @@
+# Azure Key Vault Key client library for Java
+Azure Key Vault allows you to create and store keys in the Key Vault. Azure Key Vault client supports RSA keys and elliptic curve keys, each with corresponding support in hardware security modules (HSM).
```

Comment:
Did something change or did I completely misread this the first time? This Readme looks like it is for KeyValault Keys but it's under KeyVault Secrets `keyvault/client/secrets/README.md`

Reply:
Accidentally added Keys Readme to this PR.
Thanks for the catch, reverted.